package.json

```diff
@@ -8,15 +8,19 @@
     "type": "git",
     "url": "https://github.com/AtomLinter/linter-tslint.git"
   },
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "coffeelint ./lib"
+  },
   "license": "MIT",
   "engines": {
     "atom": ">0.50.0"
   },
   "dependencies": {
-    "atom-package-deps": "^4.0.1",
+    "atom-package-deps": "4.0.1",
```

Comment:
@Arcanemagus  Let me explain some use case:
For example, the ```atom-package-deps``` has been updated to v4.1.0. v4.1.0 satisfies ```^4.0.1``` and breaks something. So until I am not fixed changes or pinned the version, all new linter-tslint users can get an error. From this point of view - strict versioning is the most stable variant.